### DISCUSSION TITLE
Deploy to Render.com button

### DISCUSSION THREAD

**Discussion by davidvm2** [Budibase/budibase#2627] [Ideas]
It would be great if you could set up a "Deploy to Render" button for quickly deploying to Render.com: https://render.com/docs/deploy-to-render

If you want to reach out to them, they're super helpful.

Retool has one already: https://render.com/docs/deploy-retool

**Comment by joebudi** [comment]
This is a great suggestion. We are focusing on our own cloud offering at the moment but will consider additional deployment options after. Thanks!

**Reply by davidvm2** [reply]
Sounds good, thanks. Any idea when cloud will be available?

**Reply by joebudi** [reply]
In a couple of weeks we'll launch the free tier of our cloud platform.

**Reply by davidvm2** [reply]
Great, thanks!
